x-pack/plugins/observability_log_explorer/README.md

```diff
@@ -1,3 +1,68 @@
 # Observability Log Explorer
 
 This plugin provides an app based on the `LogExplorer` component from the `log_explorer` plugin, but adds observability-specific affordances.
+
+## Testing
+
+### Stateful
+
+#### FTR Server
+```
+node scripts/functional_tests_server --config ./x-pack/test/functional/apps/observability_log_explorer/config.ts
+```
+
+#### FTR Runner
+```
+node scripts/functional_test_runner --config ./x-pack/test_serverless/functional/test_suites/observability/config.ts --include ./x-pack/test_serverless/functional/test_suites/observability/observability_log_explorer/index.ts
+```
+
+#### Running Individual Tests
+```
+node scripts/functional_test_runner --config ./x-pack/test_serverless/functional/test_suites/observability/config.ts --include ./x-pack/test_serverless/functional/test_suites/observability/observability_log_explorer/$1
```

Comment:
**nit:** same as the previous comment, it should point to the stateful config + why `$1`? I typically use the terminal autocomplete to add the file I want to run.

Reply:
I have a script with which i do the magic that's why i had the $1 at the end. But this represents a replaceable variable

```
ftrrunner() {
    wsk

    if [ -z "$1" ]; then
        node scripts/functional_test_runner --config ./x-pack/test/functional/apps/observability_log_explorer/config.ts --include ./x-pack/test/functional/apps/observability_log_explorer/index.ts
    else
        node scripts/functional_test_runner --config ./x-pack/test/functional/apps/observability_log_explorer/config.ts --include ./x-pack/test/functional/apps/observability_log_explorer/$1
    fi
}```